src/assets.ts

```diff
@@ -197,21 +197,17 @@ export class AssetGenerator {
 ]`;
             }
             case ProgramLaunchType.BlazorWebAssemblyHosted: {
-                const chromeLaunchConfigurationsMassaged: string = indentJsonString(createBlazorWebAssemblyLaunchConfiguration(this.computeWorkingDirectory()));
-                const hostedLaunchConfigurationsMassaged: string = indentJsonString(createBlazorWebAssemblyHostedLaunchConfiguration(this.computeProgramPath(), this.computeWorkingDirectory()));
+                const hostedLaunchConfigMassaged: string = indentJsonString(createBlazorWebAssemblyHostedLaunchConfiguration(this.computeProgramPath(), this.computeWorkingDirectory()));
                 return `
 [
-    ${hostedLaunchConfigurationsMassaged},
-    ${chromeLaunchConfigurationsMassaged}
+    ${hostedLaunchConfigMassaged}
 ]`;
             }
             case ProgramLaunchType.BlazorWebAssemblyStandalone: {
-                const chromeLaunchConfigurationsMassaged: string = indentJsonString(createBlazorWebAssemblyLaunchConfiguration(this.computeWorkingDirectory()));
-                const devServerLaunchConfigurationMassaged: string = indentJsonString(createBlazorWebAssemblyDevServerLaunchConfiguration(this.computeWorkingDirectory()));
+                const standaloneLaunchConfigMassaged: string = indentJsonString(createBlazorWebAssemblyStandaloneLaunchConfiguration());
```

Comment:
This needs to take the computed working directory doesn't it?

Reply:
Correct. I hadn't realized that the logic that automatically generated the `launch.json` file supported creating it in nested directories as well.